packages/teams-js/src/public/media.ts

```diff
@@ -106,14 +130,14 @@ export namespace media {
     if (!GlobalVars.isFramelessWindow) {
       const notSupportedError: SdkError = { errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM };
       /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-      callback(notSupportedError, undefined);
+      callback(notSupportedError, []);
```

Comment:
>[] [](http://example.com/codeflow?start=34&length=2)

In this and other places in this file, the parameter value passed in the case of an error was changed to a legal value, according to the defined types. The documentation for these callback types was also updated at the top to make it clear what the expected behavior is, in the case of an error.